### ISSUE TITLE
EDDI ISSUE - Running v.3.1.1 i have installed but each time i open EDDI it says "Only 1 instance of EDDI can run at a time" I have closed and reset computer but same message. Please can you help?

### ISSUE THREAD

**Issue by richardB123** [EDCD/EDDI#976]
## EDDI version in which issue found
[please give the full version number here, not just "latest" or "current", and also say whether you are running under Voice Attack or standalone]

## Steps to reproduce

1. [give the fullest and most reproducible steps you can]
2. [the more reproducible, the better our chances of fixing it]

## Expected
[describe what you expected to happen]

## Observed
[describe what happened instead]

## Investigation
[any investigation you have done, regression tests against earlier versions, etc]

**Comment by Tkael** [comment]
Hi, were you able to get this working?

**Comment by Muczachan** [comment]
Now I got it. It looks like EDDI thinks it is running. Even after a PC reboot. How does it  check for being the only instance? Is there a lockfile to clean or a registry entry, possibly?

**Comment by Muczachan** [comment]
Though it's on the running EDDI for the second time. The first time nothing happens, no GUI appears and I have not seen anything in the task manager. Then "1 instance" dialog when trying again.
With 3.1.2.

**Comment by Tkael** [comment]
Do you have VoiceAttack installed? Is it running when you try to open EDDI?

**Comment by Muczachan** [comment]
I do have VA installed, but I am not actually using it so it's not running. Want me to uninstall VA and try again?

**Comment by Tkael** [comment]
Sounds like there may be an issue that EDDI is encountering the first time you run it after a reboot. Please attach a log file for the first run after a reboot for review? Logs are located at `%APPDATA%/EDDI`.

**Comment by Muczachan** [comment]
[eddi.log](https://github.com/EDCD/EDDI/files/2660916/eddi.log)
Here you are.

**Comment by Tkael** [comment]
According to the log, it appears to be choking when trying to read some system information. Unfortunately, the specific system that it's choking on is not recorded. It *might* be your home system?
Go though the steps noted [here](https://github.com/EDCD/EDDI/blob/master/TROUBLESHOOTING.md#eddi-doesnt-start-or-crashes-on-startup) to determine if one of your config files is causing this?
If it's `eddi.json` (the file listing your home system / station), please send that file so that I can review the specific star system?

**Comment by Muczachan** [comment]
If we're talking about Phoo Phlai LJ-T c3-4 it was location at the time I last closed EDDI. I have moved since, but of course EDDI does not know it, since it's not running. The system most certainly exists: https://www.edsm.net/en/system/id/30587093/name/Phoo+Phlai+LJ-T+c3-4 (well, at least in that particular DB).
I'm trying the steps you have provided now. Will update.

**Comment by Muczachan** [comment]
It's the credentials.json that's causing the problem.

**Comment by Muczachan** [comment]
And I have problems logging into Frontier API. I did get the email code alright, but no luck connecting. May be external problem?

**Comment by Muczachan** [comment]
And yes. After the Frontier API connection failed, I was unable to start EDDI again until I have removed the credentials.json.

**Comment by Tkael** [comment]
Acknowledged. Without credentials.json it succeeds?

**Comment by Muczachan** [comment]
Yes it does.

**Comment by Tkael** [comment]
I think I've got it... EDDI's choking on null data because the system it thinks you are in isn't found in the old server data. 
The next version will be more tolerant of null data from the old legacy server.

**Comment by richardbuckle** [comment]
Is this an "already fixed", @Tkael ?

**Comment by Tkael** [comment]
Already fixed on my branch. This ticket will be closed when my branch is merged.

**Comment by Muczachan** [comment]
I am still unable to log into Frontier API. Shall I create a separate ticket for it?

**Comment by Tkael** [comment]
Let's wait on that until after chapter 4 release. FDev have talked about adopting a new authentication system for 3.3. If they turn off the old server immediately we may need to rewrite that section of code sooner rather than later anyways.

**Comment by Muczachan** [comment]
Fair enough, I'm 17k ly outband from Sol anyway, not like I'll be in need of market info anytime soon. Quite probably for months.
